tests/integration/test_diags.py

```diff
@@ -82,7 +82,7 @@ def compare_images(
     expected_png = Image.open(path_to_expected_png).convert("RGB")
     diff = ImageChops.difference(actual_png, expected_png)
 
-    diff_dir = "image_check_failures"
+    diff_dir = f"{TEST_ROOT_PATH}image_check_failures"
```

Comment:
The `image_check_failures/` dir was being placed in the root of the repo. `.gitignore` wasn't ignoring it as a result.